### ISSUE TITLE
Updating FluentTabs ActiveTabId doesn't rerender if data is entered in contained FluentTextInput

### ISSUE THREAD

**Issue by sschutten** [microsoft/fluentui-blazor#1314]
# 🐛 Bug Report

If a FluentTab contains a FluentTextField and a user enters some text in the text field, the FluentTabs doesn't rerender if the ActiveTabId is updated. This is broken between 4.2.1 and 4.3.0. I guess this is related to the fix for https://github.com/microsoft/fluentui-blazor/issues/1194.

## 💻 Repro or Code Sample

Given the snippet below, perform the following steps:
- Enter some text in the text field on the Foo tab
- Click the Next button

Home.razor
``` razor
<FluentTabs ActiveTabId="@_selectedTabId">
    <FluentTab Label="Foo" Id="foo">
        <FluentTextField Label="Foo" />
    </FluentTab>
    <FluentTab Label="Bar" Id="bar">
        Bar
    </FluentTab>
</FluentTabs>

<FluentButton OnClick="Next">Next ></FluentButton>

@code {
    private string _selectedTabId = "foo";

    private void Next()
    {
        _selectedTabId = "bar";
    }
}
```

## 🤔 Expected Behavior

The Bar tab is shown.

## 😯 Current Behavior

The Foo tab remains visible.

## 🌍 Your Environment

This is working on version 4.2.1 and is broken as of version 4.3.0.

**Comment by vnbaaij** [comment]
Hi Sander,

Yes, this is indeed caused by the fix for #1194. But I've found a solution...

In next release (v4.3.1) we will add a new method `GoToTab(string TabId)` on `FluentTabs` component. To call the method you will need to use a `@ref`. Your code would look like this:
```
<FluentTabs @ref="tabs" @bind-ActiveTabId="@_selectedTabId">
    <FluentTab Label="Foo" Id="foo">
        <FluentTextField Label="Foo" />
    </FluentTab>
    <FluentTab Label="Bar" Id="bar">
        Bar
    </FluentTab>
</FluentTabs>

<FluentButton OnClick="Next">Next ></FluentButton>

@code {
    private FluentTabs? tabs;
    private string _selectedTabId = "foo";

    private async Task Next()
    {
        await tabs!.GoToTab("bar");
    }
}
```
The @bind-ActiveTabId can be left out but you will get a weird glitch on the first tab change (the web component and the Blazor component are 'out of sync'). Just using `ActiveTabId` does not work at all. Deemded that of lower priority so have not tracked that down yet.

**Comment by sschutten** [comment]
Thanks, this sounds like a workable solution. We'll stick to 4.2.1 until 4.3.1 is released, which is fine.
